packages/realm_generator/test/error_test_data/invalid_model_name_mapping.dart

```diff
@@ -1,10 +0,0 @@
-import 'package:realm_common/realm_common.dart';
-//part 'invalid_model_name_mapping.g.dart';
-
-const one = '1';
-
-@RealmModel()
-@MapTo(one) // <- invalid
-// prefix is not important, as we explicitly define name with @MapTo,
-// but obviously 1 is not a valid class name
```

Comment:
Is this removed because `'1'` is now a valid class name?

Reply:
No. But this test is not actually run on main either. Apparently it broke about the time when the generator tests was refactored to use `.expected` files.

The reason is this check:
```dart
    if (_path.extension(file.path) == '.dart' && !file.path.endsWith('g.dart')) {
```
should probably have been:
```dart
    if (_path.extension(file.path) == '.dart' && !file.path.endsWith('.g.dart')) {
```
or just
```dart
    if (_path.extension(file.path) == '.dart') {
```
since there are no `.g.dart` file in those directories.

So, I'm just making it explicit that the test is not being run. It would fail, if we fix the mistake. 

The test originates back to a time when `@MapTo` on classes had a different meaning than today. Is `1` actually an illegal class name in realm-core?

Reply:
I think it should be - at least haven't seen any restrictions on numbers being used as table names.